Applications/GesturePod/README.md

```diff
@@ -8,7 +8,7 @@ for instance a mobile phone. Watch the [GesturePod video here](https://1drv.ms/u
 
 In this directory, we provide resources for
   1. Creating a GesturePod of your own based on the [Arduino MKR1000](https://store.arduino.cc/usa/arduino-mkr1000), in the `onMKR1000` directory, and
-  2. Running a simulation of the gesture recognition algorithm on your desktop in the `onWorkStation` directory. 
+  2. Running a simulation of the gesture recognition algorithm on your desktop in the `onComputer` directory. 
```

Comment:
The bare-bones gesture recognition pipeline using saved data. (please don't use the simulation word)

Reply:
replaced with Gesture Recognition Pipeline